plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/flex/CreateAppEngineFlexWtpProject.java

```diff
@@ -66,19 +66,26 @@
   private static final List<MavenCoordinates> PROJECT_DEPENDENCIES;
 
   static {
-    MavenCoordinates.Builder servletApi = new MavenCoordinates.Builder()
+    // FIXME: servlet-api and jsp-api should be provided by the servlet container and not included
+    MavenCoordinates servletApi = new MavenCoordinates.Builder()
         .setGroupId("javax.servlet") //$NON-NLS-1$
         .setArtifactId("javax.servlet-api") //$NON-NLS-1$
-        .setVersion("3.1.0"); //$NON-NLS-1$
-    PROJECT_DEPENDENCIES = Collections.singletonList(servletApi.build());
+        .setVersion("3.1.0") //$NON-NLS-1$
+        .build();
```

Comment:
I was going to say that, since `javax.servlet-api` is being not exported, what you add here would not be exported either. We want to export JSTL, right?

Reply:
We do.  I'll bite the bullet and put the JSTL library into `WEB-INF/lib`.